Resources/Prototypes/DeltaV/Catalog/Fills/Lockers/clerk.yml

```diff
@@ -0,0 +1,14 @@
+- type: entity
+  parent: LockerClerk
+  id: LockerClerkFilled
+  name: clerk's locker
```

Comment:
![image](https://github.com/DeltaV-Station/Delta-v/assets/113523727/9581fff0-28a7-4181-ac6f-4fab4ff9c442)

this shares the same name as the empty one without any suffix, making it confusing for mappers